knowledge_repo/app/templates/markdown-base.html

```diff
@@ -15,6 +15,11 @@
             <button type="button" class="btn btn-default btn-raw">
               View Raw Markdown
             </button>
+            {% if is_author and webeditor_buttons %}
```

Comment:
nit: I think the server should completely control the logic of whether or not to show the button - so I'd have this bit of logic live in render.py